ui/app/components/app/asset-list-item/asset-list-item.js

```diff
@@ -40,6 +58,36 @@ const AssetListItem = ({
     )
     : null
 
+  const sendTokenButton = useMemo(() => {
+    if (tokenSymbol === 'ETH') {
+      return null
+    }
+    return (
+      <Button
+        type="link"
+        className="asset-list-item__send-token-button"
+        onClick={(e) => {
+          e.stopPropagation()
+          sendTokenEvent()
+          dispatch(updateSendToken({
+            address: tokenAddress,
```

Comment:
It's important to pass in the decimals here as well. They're used in the send flow.

Reply:
It looks like the `tokenData` object being passed to `TokenCell` already has `decimals` in it, but `TokenCell` isn't using it as a prop yet. It needs to be added, and passed along here to `AssetListItem`. Or you could assemble it back into a `token` object in `TokenCell` to pass down here.

Unfortunately the type for "token" varies a great deal. There are three properties used everywhere (`address`, `symbol`, and `decimals`, where only `address` is required), but the balance is present sometimes as well under different names and in different formats. I think we only want the three-property version here though. All uses of `sendToken` properties I can find are one of those three.